infra/main.bicep

```diff
@@ -210,6 +210,9 @@ param logLevel string = 'INFO'
 @description('List of comma-separated languages to recognize from the speech input. Supported languages are listed here: https://learn.microsoft.com/en-us/azure/ai-services/speech-service/language-support?tabs=stt#supported-languages')
 param recognizedLanguages string = 'en-US,fr-FR,de-DE,it-IT'
 
+@description('Health check path for the web app')
+param webHealthCheckPath string = '/api/health'
```

Comment:
I'm thinking we should just hard code this given we're not updating the path used by the app using this.